### ISSUE TITLE
Show more stats

### ISSUE THREAD

**Issue by jtpio** [voila-gallery/voila-spotify#2]
It would be nice to extend the dashboard with a bunch of other stats, such as:

- number of unique songs
- total minutes played per day / week / month / year
- favorite artists / tracks
- cycles (when a track was listened to the last time), maybe by drawing a line or arc from the previous day to the current day

I guess the Spotify "Year in Music" does a similar thing, but it would be cool to have it anyway here too :)

**Comment by martinRenou** [comment]
Yes! I would love that :) If we could fetch favorite artist's images that would put some color in the dashboard as well. I wonder if there is a Spotify API we can use for that.

**Comment by jtpio** [comment]
There is one but it requires a token. Not sure if there are some public endpoints though.
